prow/README.md

```diff
@@ -31,6 +31,12 @@ state and no claims of backwards compatibility are made for any external API.
    Cluster administrators upgrading to the newest version of Prow should move
    plugin configuration from the main `ConfigMap`. For more context, please see
    [this pull request.](https://github.com/kubernetes/test-infra/pull/4213)
+ - *September 1, 2017* Deck and Jenkins-Operator controllers no longer provide
```

Comment:
These should go the other way, with newer entries at the top.

Reply:
Can we also include the versions or commits that break?